### PR TITLE
Add https to all links to make them hyperlinks

### LINKED ISSUE #21: Make project urls hyperlinks
When adding the pvOps project in #18, I included https in the links to the project and documentation (see screenshot below). This caused the links to be represented as hyperlinks, which I think is desirable. 

I suggest adding http/https to all of the URLs to make the readthedocs website more user-friendly.

An alternative solution could be to make the links hyperlinks without displaying HTTP, but I don't have a solution for this.

image

### PR DESCRIPTION
**Pull request by AdamRJensen**
Closes #21

### CODE REVIEW

**Comment by AdamRJensen** [comment]
I checked all links to ensure they work and updated a number of documentation websites. See the commit descriptions above.

**Comment by AdamRJensen** [comment]
@kandersolar Ready for review!

File: openpvtools.csv [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-twoaxistracking,Two-axis tracker shading,2021 - *,twoaxistracking.readthedocs.io,github.com/pvlib/twoaxistracking,Python,BSD 3
+PVLib Matlab,General purpose PV modeling,2012? - *,https://pvpmc.sandia.gov,https://github.com/sandialabs/MATLAB_PV_LIB,Matlab,BSD 3
+PVLib Python,General purpose PV modeling,2013 - *,https://pvlib-python.readthedocs.io,https://github.com/pvlib/pvlib-python,Python,BSD 3
+System Advisor Model (SAM),"Desktop app for PV, wind, CSP modeling, financial",2004 - *,https://sam.nrel.gov,https://github.com/NREL/SAM,C++,Mixed MIT/GPL 3
+ssc,Compute modules for SAM,2004 - *,https://sam.nrel.gov,https://github.com/nrel/ssc,"C, C++",Mixed MIT/GPL 3
+RdTools,Technical analysis of PV timeseries data,2016 - *,https://rdtools.readthedocs.io,https://github.com/NREL/rdtools,Python,MIT
+PVFree,API for obtaining PV modeling parameters,2015 - *,https://pvfree.azurewebsites.net,https://github.com/BreakingBytes/pvfree,Python,BSD 2
+SolarUtils,Python wrappers of C solar position and spectral decomposition,2016,https://github.com/SunPower/SolarUtils,https://github.com/SunPower/SolarUtils,Python,BSD 3
```
Comment (kandersolar) [inline openpvtools.csv]:
```suggestion
SolarUtils,Python wrappers of C solar position and spectral decomposition,2016,https://sunpower.github.io/SolarUtils/,https://github.com/SunPower/SolarUtils,Python,BSD 3
```

File: openpvtools.csv [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+Pecos,Performance monitoring,2016 - *,https://pecos.readthedocs.io,https://github.com/sandialabs/pecos,Python,BSD 3
+Solpy,General purpose PV modeling,2011-2015,https://solpy.readthedocs.io,https://github.com/nrcharles/solpy,Python,LGPL 2.1
+PVMismatch,IV curve calculator for mismatched cells,2012 - *,https://sunpower.github.io/PVMismatch/,https://github.com/SunPower/PVMismatch,Python,BSD 3
+photovoltaic,General purpose PV modeling,2017 - *,https://github.com/trautsned/photovoltaic,https://github.com/trautsned/photovoltaic,Python,GPL 3
```
Comment (kandersolar) [inline openpvtools.csv]:
```suggestion
photovoltaic,General purpose PV modeling,2017 - *,https://github.com/pvedu/photovoltaic,https://github.com/pvedu/photovoltaic,Python,GPL 3
```

File: openpvtools.csv [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+feedinlib,PV timeseries modeling,2015 - *,https://feedinlib.readthedocs.io,https://github.com/oemof/feedinlib,Python,GPL 3
+CASSYS,PV system modeling,2015 - *,https://github.com/CanadianSolar/CASSYS,https://github.com/CanadianSolar/CASSYS,"Excel, C#",BSD 3
+Bifacial PV View Factor model,Bifacial modeling,2017 - *,https://bifacialvf.readthedocs.io,https://github.com/NREL/bifacialvf,Python,Unknown
+solaR,General purpose PV modeling,2010 - *,https://jstatsoft.org/article/view/v050i09,https://github.com/oscarperpinan/solar,R,GPL 3
+pvfactors,Diffuse shading and bifacial irradiance modeling,2016 - *,https://sunpower.github.io/pvfactors/,https://github.com/SunPower/pvfactors,Python,BSD 3
+pvcaptest,Capacity testing per ASTM E2848,2017 - ,https://pvcaptest.readthedocs.io/,https://github.com/pvcaptest/pvcaptest,Python,MIT
+SolarData,Accessing public solar datasets,2018 - ,https://github.com/dazhiyang/SolarData,https://github.com/dazhiyang/SolarData,R,GPL 2
+SolCore,modelling solar cells and semiconductor materials,2017 -- *,https://solcore.solar,https://github.com/qpv-research-group/solcore5,Python,LGPL 3
```
Comment (kandersolar) [inline openpvtools.csv]:
```suggestion
SolCore,modelling solar cells and semiconductor materials,2017 -- *,https://www.solcore.solar,https://github.com/qpv-research-group/solcore5,Python,LGPL 3
```
the www seems to be necessary

**Review by kandersolar — APPROVED** [review]

**Pull request merged**